wallet/tickets.go

```diff
@@ -48,9 +40,19 @@ func (w *Wallet) LiveTicketHashes(rpcClient *chain.RPCClient, includeImmature bo
 	// This was mostly copied from an older version of the legacy RPC server
 	// implementation, hence the overall weirdness, inefficiencies, and the
 	// direct dependency on the consensus server RPC client.
+	type promiseGetTxOut struct {
+		result dcrrpcclient.FutureGetTxOutResult
+		ticket chainhash.Hash
```

Comment:
I would probably make this a pointer and point into the slice, reduces some copying